exdb/views.py

```diff
@@ -49,10 +48,12 @@ class HallStaffDashboardView(TemplateView):
 
     def get_context_data(self):
         context = super(HallStaffDashboardView, self).get_context_data()
-        context['pending_experiences'] = Experience.objects.filter(status='pe')
-        context['experiences_needing_eval'] = [
-            e for e in Experience.objects.filter(
-                status='ad') if e.needs_evaluation()]
+        context['user'] = get_user_model().objects.prefetch_related('approval_queue__author',
+                                                                    'approval_queue__keywords',
+                                                                    'approval_queue__recognition',
+                                                                    'approval_set__experience__author',
+                                                                    'approval_set__experience__keywords',
+                                                                    'approval_set__experience__recognition').get(pk=self.request.user.pk)
```

Comment:
Do this
```python
context['user'] = get_user_model().objects.prefetch_related(
    'approval_queue__author',
    'approval_queue__keywords',
    'approval_queue__recognition',
    'approval_set__experience__author',
    'approval_set__experience__keywords',
    'approval_set__experience__recognition'
).get(pk=self.request.user.pk)
```
It is more consistent with the way we have handled large queries in the past.